src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/management/ShortlinkTestsIT.java

```diff
@@ -0,0 +1,146 @@
+  }
+  
+  @Test
+  public void testFindShortlinks() {
+    String organizationId = getOrganizationId(0);
+    given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/shortlinks/{shortlinkId}", organizationId, getOrganizationShortlinkId(organizationId, 0))
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id", notNullValue())
+      .body("path", is("/elsewhere"))
+      .body("name", is("Elsewhere"));
+  } 
+  
+  @Test
+  public void testListShortlinksBySlug() {
+    given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/shortlinks?path=/hills", getOrganizationId(0))
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id.size()", is(1))
+      .body("id[0]", notNullValue())
+      .body("path[0]", is("/hills"));
+    
+    given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/shortlinks?path=non-existing", getOrganizationId(0))
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id.size()", is(0));
+  } 
+  
+  @Test
+  public void testListShortlinks() {
+    given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/shortlinks", getOrganizationId(0))
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id.size()", is(3))
+      .body("id[1]", notNullValue())
+      .body("path[1]", is("/beyond"))
+      .body("name[1]", is("Beyond"));
+  } 
+  
+  @Test
+  public void testOrganizationShortlinksNotFound() throws InterruptedException {
+    String organizationId = getOrganizationId(0);
+    String incorrectOrganizationId = getOrganizationId(1);
+    String organizationShortlinkId = getOrganizationShortlinkId(organizationId, 0);
+    
+    String[] malformedIds = new String[] {"evil", "*", "/", "1", "-1", "~"};
+    assertFound(String.format("/organizations/%s/shortlinks/%s", organizationId, organizationShortlinkId));
+    assertEquals(3, countApiList(String.format("/organizations/%s/shortlinks", organizationId)));
+    
+    for (String malformedId : malformedIds) {
+      assertNotFound(String.format("/organizations/%s/shortlinks/%s", organizationId, malformedId));
+    }
+    
+    assertNotFound(String.format("/organizations/%s/shortlinks/%s", incorrectOrganizationId, organizationShortlinkId));
+    assertEquals(0, countApiList(String.format("/organizations/%s/shortlinks", incorrectOrganizationId)));
+  }
+
+  private void createPtvSettings() {
+    insertSystemSetting(PtvConsts.SYSTEM_SETTING_BASEURL, String.format("%s%s", getWireMockBasePath(), BASE_URL));
+    flushCache();
+  }
+  
+  private void deletePtvSettings() {
+    deleteSystemSetting(PtvConsts.SYSTEM_SETTING_BASEURL);
+  }
+    
+  private void createManagementSettings(String organizationId) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "createManagementSettings" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)